extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -116,12 +130,14 @@ export class IframeTransport {
       sentinel,
       type: this.type_,
     }));
-    const frame = createElementWithAttributes(this.ampWin_.document, 'iframe',
-        /** @type {!JsonObject} */ ({
-          sandbox: 'allow-scripts allow-same-origin',
-          name: frameName,
-          'data-amp-3p-sentinel': sentinel,
-        }));
+
+    const frame = /** @type {!HTMLIFrameElement} */
+        (createElementWithAttributes(this.ampWin_.document, 'iframe',
+            /** @type {!JsonObject} */ ({
```

Comment:
You can avoid the cast here by using dict().

Reply:
Done